command/rootchain/deploy/deploy.go

```diff
@@ -378,31 +378,33 @@ func runCommand(cmd *cobra.Command, _ []string) {
 		return
 	}
 
-	outputter.SetCommandResult(&helper.MessageResult{
+	outputter.WriteCommandResult(&helper.MessageResult{
 		Message: fmt.Sprintf("%s finished. All contracts are successfully deployed and initialized.",
 			contractsDeploymentTitle),
 	})
+	outputter.SetCommandResult(command.Results(commandResults))
 }
 
 // deployContracts deploys and initializes rootchain smart contracts
 func deployContracts(outputter command.OutputFormatter, client *jsonrpc.Client, chainID int64,
-	initialValidators []*validator.GenesisValidator, cmdCtx context.Context) (*polybft.RootchainConfig, int64, error) {
+	initialValidators []*validator.GenesisValidator, cmdCtx context.Context) (
+	*polybft.RootchainConfig, int64, []command.CommandResult, error) {
```

Comment:
Can we introduce some struct which would have the following fields and return instance of that struct instead:
- `rootchainCfg`
- `chainID`
- `commandResults`.

You can call it `deploymentResultInfo` (or if you have some better name feel free to name it that way :wink:).